README.md

```diff
@@ -1 +1,40 @@
-# Decode-morse-code
\ No newline at end of file
+# Morse code decodng.
+
+This is a solution for morse-code decoding.
+
+## Authors
+
```

Comment:
### Optional
- [ ] It would be better if we have the getting started column in our README file. Here is our [template](https://github.com/microverseinc/readme-template/blob/master/README.md)